test/graphqlQueries.ts

```diff
@@ -781,3 +781,92 @@ export const editProjectUpdateQuery = `
+           createProjectVerificationForm(projectId: $projectId) {
+                    id
+                    user {
+                      id
+                    }
+                    project {
+                      id
+                    }
+                    status
+                    }
+            }
+        `;
+
+export const getCurrentProjectVerificationFormQuery = `
+        query getCurrentProjectVerificationForm($projectId: Float!){
+           getCurrentProjectVerificationForm(projectId: $projectId) {
+                    id
+                    user {
+                      id
+                    }
+                    project {
+                      id
+                    }
+                    status
+                  }
+            }
+        `;
+
+export const updateProjectVerificationFormMutation = `
+        mutation updateProjectVerificationForm($projectVerificationUpdateInput: ProjectVerificationUpdateInput!){
+           updateProjectVerificationForm(projectVerificationUpdateInput: $projectVerificationUpdateInput) {
+                    id
+                    user {
+                      id
+                    }
+                    project {
+                      id
+                    }
+                    status
+                    }
+            }
+        `;
+
+// export const updateProjectVerificationFormMutation = `
+//         mutation updateProjectVerificationForm($projectVerificationUpdateInput: ProjectVerificationUpdateInput!){
+//            updateProjectVerificationForm(projectVerificationUpdateInput: $projectVerificationUpdateInput) {
+//                     id
+//                     user {
+//                       id
+//                     }
+//                     project {
+//                       id
+//                     }
+//                     status
+//                     }
+//                     projectContacts {
+//                       youtube
+//                       instagram
+//                       linkedin
+//                       facebook
+//                       instagram
+//                     }
+//                     projectRegistry {
+//                       organizationDescription
+//                       isNonProfitOrganization
+//                       organizationCountry
+//                       organizationWebsite
+//                     }
+//
+//                     milestones {
+//                       mission
+//                       foundationDate
+//                       achievedMilestones
+//                       achievedMilestonesProof
+//                     }
+//                     managingFunds {
+//                       description
+//                       relatedAddress {
+//                         address
+//                         networkId
+//                         title
+//                       }
+//                     }
+//
+//             }
+//         `;
```

Comment:
@CarlosQ96 
I mean this, when you want to get `managingFunds` or `milestones` or `projectRegistry` or `projectContacts` you get `400` error